src/AnonymizationStep/AnonymizationStep.tsx

```diff
@@ -42,42 +42,45 @@ const emptySummary: AnonymizationSummary = {
+  return (
+    <Descriptions className="AnonymizationSummary-descriptions" layout="vertical" bordered column={{ sm: 2, md: 4 }}>
+      <Descriptions.Item label="Suppressed Count">
+        {`${summary.lowCountRows} of ${summary.totalRows} (${formatPercentage(
+          summary.lowCountRows / summary.totalRows,
+        )})`}
+      </Descriptions.Item>
+      <Descriptions.Item label="Suppressed Bins">
+        {`${summary.lowCountBuckets} of ${summary.totalBuckets} (${formatPercentage(
+          summary.lowCountBuckets / summary.totalBuckets,
+        )})`}
+      </Descriptions.Item>
+      <Descriptions.Item label="Median Distortion">{formatPercentage(summary.medianDistortion)}</Descriptions.Item>
+      <Descriptions.Item label="Maximum Distortion">{formatPercentage(summary.maxDistortion)}</Descriptions.Item>
+    </Descriptions>
+  );
 }
 
 function AnonymizationSummary({ result: { summary }, loading }: CommonProps) {
   return (
-    <div className="AnonymizationSummary loading notebook-step">
+    <>
       <NotebookNavAnchor step={NotebookNavStep.AnonymizationSummary} status={loading ? 'loading' : 'done'} />
       <Title level={3}>Anonymization summary</Title>
-      <Descriptions className="AnonymizationSummary-descriptions" layout="vertical" bordered column={{ sm: 2, md: 4 }}>
-        <Descriptions.Item label="Suppressed Count">
-          {!loading ? (
-            `${summary.lowCountRows} of ${summary.totalRows} (${formatPercentage(
-              summary.lowCountRows / summary.totalRows,
-            )})`
-          ) : (
-            <TextPlaceholder />
-          )}
-        </Descriptions.Item>
-        <Descriptions.Item label="Suppressed Bins">
-          {!loading ? (
-            `${summary.lowCountBuckets} of ${summary.totalBuckets} (${formatPercentage(
-              summary.lowCountBuckets / summary.totalBuckets,
-            )})`
-          ) : (
-            <TextPlaceholder />
-          )}
-        </Descriptions.Item>
-        <Descriptions.Item label="Median Distortion">
-          {!loading ? formatPercentage(summary.medianDistortion) : <TextPlaceholder />}
-        </Descriptions.Item>
-        <Descriptions.Item label="Maximum Distortion">
-          {!loading ? formatPercentage(summary.maxDistortion) : <TextPlaceholder />}
-        </Descriptions.Item>
-      </Descriptions>
-    </div>
+      {summary === emptySummary ? (
+        <div className="AnonymizationSummary loading notebook-step">
+          <Space direction="vertical">
+            <Spin size="large" />
+            <Text type="secondary">Anonymizing data</Text>
+          </Space>
+        </div>
+      ) : loading ? (
+        <div className="AnonymizationSummary completed notebook-step">
+          <Spin>{summaryDescriptions(summary)}</Spin>
+        </div>
+      ) : (
+        <div className="AnonymizationSummary completed notebook-step">{summaryDescriptions(summary)}</div>
+      )}
```

Comment:
Can be condenset to `<Spin spinning={loading}>summaryDescriptions(summary)}</Spin>`